apis/api-directory/cloud-cost-api.md

```diff
@@ -0,0 +1,53 @@
+# Cloud Cost API
+
+{% swagger method="get" path="query/cloudcost/query" baseUrl="http://app.kubecost.com/model" summary="Cloud Cost API (Querying)" %}
+{% swagger-description %}
+The Cloud Cost API is the preferred way to query for costs and resources related to cloud provider services.
+{% endswagger-description %}
+
+{% swagger-parameter in="path" name="window" required="true" type="string" %}
+Duration of time over which to query. Accepts multiple formats including units of time, relative time units, and unix timestamps. See this section on [using the `window` parameter](/apis/api-directory/api-directory.md#using-the-window-parameter) for more information.
+{% endswagger-parameter %}

```

Comment:
Having some problems generating data when querying for roughly less than 12 hours. Returns a 200 but no data

Reply:
This seems to also apply to Allocation/Assets, sampling under 24h doesn't return any data

Reply:
The resolution for cloud costs is 24h, but you should be able to get data for assets/allocations if querying under 24h. I'll take a look at this :)